Cargo.toml

```diff
@@ -1,17 +1,28 @@
 [package]
 name = "netcdf"
-version = "0.1.0"
-authors = ["Michael Hiley <mike.hiley@ssec.wisc.edu>"]
+version = "0.3.2"
+authors = [
+    "Michael Hiley <mike.hiley@ssec.wisc.edu>",
+    "Magnus Ulimoen <magnusu@met.no>"
+]
 license = "MIT OR Apache-2.0"
 description = "High-level NetCDF bindings for Rust"
 repository = "https://github.com/mhiley/rust-netcdf"
 documentation = "https://github.com/mhiley/rust-netcdf"
 keywords = ["netcdf", "hdf", "hdf5", "libnetcdf", "netcdf4"]
+edition = "2018"
+
+[features]
+default = ["ndarray", "indexmap"]
+memory = ["netcdf-sys/memio"]
 
 [dependencies]
-libc = "0.2"
-lazy_static = "1.0"
-ndarray = "0.11.0"
+lazy_static = "1.4.0"
+ndarray = { version = "0.12.*", optional = true }
```

Comment:
ndarray 0.13 is the latest as of this writing (I don't think there will be any code to update, for that).